.git-deps.yml

```diff
@@ -0,0 +1,128 @@
+deps:
+# COMMON
+- name: pmm
+  branch: PMM-2.0
+  path: sources/pmm/src/github.com/percona/pmm
+  url: https://github.com/percona/pmm
+  component: common
+# CLIENT
+- name: pmm-admin
```

Comment:
Could we put an empty line before each new `-name` section to make it more readable?